tests/integration/pilot/webhook_test.go

```diff
@@ -91,6 +94,38 @@ func TestWebhook(t *testing.T) {
 		})
 }
 
+// Verify that the controller validation webhook service is reachable at
+// https://IstioPodIP:15017/validate . This is being explicitly done
+// to make sure, in dual-stack scenarios both v4 and v6 endpoints are reachable.
+func TestWebhookValidationEndpoints(t *testing.T) {
```

Comment:
Hmmm. This would probably not work well with external istiod. Is this not tested by applying a config which will already have the control plane call istiod?

Reply:
L155 already does a  verifyRejectsInvalidConfig(). But in dual-stack scenario, it is always going to the default ip family. So felt it is better to have an explicit test to make sure both the endpoints are reachable in dual stack scenario. If needed, we can skip this test for non-dual-stack cases. let me know.

Reply:
If its always going to the default IP family in real world, why do we need to test both?

Reply:
@zhlsunshine : Could you please confirm if the validation always goes to the default ip family endpoint? In that case probably this additional test is not needed?